File: cookbook/python/automations/endpoints/update.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.local")

reclient = Retab()

automation = reclient.processors.automations.endpoints.update(
    endpoint_id="endp_jKH0WW6X1dD3wtWbnuuXQ",
    name="Endpoint Automation Updated",
)

print(automation.model_dump_json(indent=2))

File: cookbook/node/automations/links/update.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

config({ path: '../../../.env.production' });

const reclient = new Retab();

const automation = await reclient.processors.automations.links.update({
  link_id: "lnk_Xf15nXFpo7mwfGT1aSYo4",
  name: "Link Automation Updated",
});

console.log(JSON.stringify(automation, null, 2));